charts/kubernetes-agent/package.json

```diff
@@ -3,5 +3,11 @@
   "version": "0.7.1",
   "private": true,
   "description": "The Octopus Kubernetes Agent",
-  "author": "Octopus Deploy Ptd Ltd"
+  "author": "Octopus Deploy Ptd Ltd",
+  "scripts": {
+    "test": "cross-env-shell docker run -ti --rm -v $INIT_CWD:/apps helmunittest/helm-unittest:latest ."
```

Comment:
I can't run the unittest code in windows, so this just runs it in a docker container. just has to be executed via `npm test`